sdk/eventhub/azure-eventhub/azure/eventhub/aio/_client_base_async.py

```diff
@@ -480,10 +466,11 @@ async def _close_connection_async(self) -> None:
         await self._client._conn_manager_async.reset_connection_if_broken()  # pylint:disable=protected-access
 
     async def _handle_exception(self, exception: Exception) -> Exception:
-        if not self.running and isinstance(exception, compat.TimeoutException):
-            exception = errors.AuthenticationException("Authorization timeout.")
-            return await _handle_exception(exception, self)
-
+        if not self.running and isinstance(exception, TimeoutError):
+            exception = errors.AuthenticationException(
```

Comment:
Why would we be raising pyamqp errors from EH code? Or is this not exposed to customers?

Reply:
it's not exposed to customers and is handled within retry, we raise amqp errors because we would like to reuse current retry logic which takes action (close connection) based on amqp error type.